raiden/raiden_service.py

```diff
@@ -241,6 +241,9 @@ def __init__(self, chain, default_registry, private_key_bin, transport, discover
         self.alarm = AlarmTask(chain)
         self.shutdown_timeout = config['shutdown_timeout']
         self._block_number = None
+        self.shutting_down_event = Event()
+        self.initialization_complete_event = Event()
```

Comment:
Can't we use the names `start_event` and `stop_event` to be consistent with the protocol and the methods `stop` and `start`?

Reply:
sure